server/http/routes/verifiers.ts

```diff
@@ -0,0 +1,182 @@
+import * as COMMON from '@dpo-packrat/common';
+import * as H from '../../utils/helpers';
+import * as LOG from '../../utils/logger';
+import * as WFV from '../../workflow/impl/Packrat/WorkflowVerifier';
+import * as DBAPI from '../../db';
+import { ASL, LocalStore } from '../../utils/localStore';
+
+export async function routeRequest(request: Request, response: Response): Promise<void> {
+
+    const verifierToRun = request.params.id;
+    console.warn(verifierToRun+'|'+JSON.stringify(request.params));
+
+    // if nothing then complain
+    if(verifierToRun===undefined) {
+        LOG.error('HTTP request: incorrect usage of endpoint', LOG.LS.eHTTP);
+        response.send('Request failed. Incorrect use of endpoint. Be sure to include which verifier to use.');
+        return;
+    }
+
+    // handle the proper type
+    switch(verifierToRun){
+        case 'edan': {
+            return await verifyEdanWorkflow(request,response);
+        } break;
+
+        default: {
+            LOG.error(`HTTP request: unsupported verify type (${verifierToRun})`, LOG.LS.eHTTP);
+            response.send(`Request failed. Unsupported verify type/path (${verifierToRun})`);
+        }
+    }
+}
+
+// TODO: progressively build up report so that if requested before done it returns partial results
+//       requires changing verifier to append after each subject, connecting tightly to workflow logic
+// TODO: fork verifier(s) so it does not use the same event loop as the main server improving performance
+//       https://nodejs.org/api/child_process.html
+// TODO: support server side events (SSE) to provide notifications to client on progress
+async function verifyEdanWorkflow(req: Request, response: Response): Promise<void> {
+    LOG.info('(Workflows) Verifying EDAN Records from endpoint...', LOG.LS.eGQL);
+
+    const workflowEngine: WF.IWorkflowEngine | null = await WF.WorkflowFactory.getInstance();
+    if (!workflowEngine) {
+        const error: string = 'verifiers createWorkflow could not load WorkflowEngine';
+        sendResponseMessage(response,false,error);
+        return;
+    }
+
+    // create our workflow (but don't start) and add it to the DB
+    const wfParams: WF.WorkflowParameters = {
+        eWorkflowType: COMMON.eVocabularyID.eWorkflowTypeVerifier,
+        //idSystemObject: undefined, // not operating on SystemObjects
+        //idProject: TODO: populate with idProject
+        //idUserInitiator: this.user?.idUser,   // not getting user at this point (but should when behind wall)
+        autoStart: false // don't start the workflow because we need to configure it
+    };
+    const workflow: WF.IWorkflow | null = await workflowEngine.create(wfParams);
+    if (!workflow) {
+        const error: string = `unable to create EDAN Verifier workflow: ${H.Helpers.JSONStringify(wfParams)}`;
+        sendResponseMessage(response,false,error);
+        return;
+    }
+
+    // grab our config options from query params

```

Comment:
I was thinking this could be a default property vs. something specific to just the EDAN verifier. I'll look at how WorkflowEngine references this and update.

Reply:
The key point here is that the workflow parameters exist to supply standard params, plus workflow-specific params.  You can and should take advantage of that to pass all of the params into the workflow, instead of having to create it, then configure it, then start it.  Just create it with the required params!  If you do this, then you can probably autoStart: true it.